Resources/public/js/views/ez-universaldiscoveryview.js

```diff
@@ -54,7 +54,7 @@ YUI.add('ez-universaldiscoveryview', function (Y) {
                 }
             });
             this.after('universalDiscoverySelectedView:contentStructChange', function (e) {
-                e.target.set('confirmButtonEnabled', !this._isAlreadySelected(e.newVal));
```

Comment:
I realize it was a mistake to write that. Instead the universal discovery methods should receive a `isAlreadySelected` function (which expects a content struct, something like `Y.bind(this._isAlreadySelected, this)` in the current code) so that each method can check the selection or pass this function to a sub-view which can then check the selection. That way, the universalDiscoverySelectedView can decide on its own the status of the confirm button.
Once this change made, pretty much the same strategy should be applied to `isSelectable`. It should be passed to the methods so that when `multiple` is set to true, the method can check if the content is OK.